src/Sylius/Component/Core/Handler/CodeChangeHandlerInterface.php

```diff
@@ -9,14 +9,14 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Component\Core\Locale\Handler;
+namespace Sylius\Component\Core\Handler;
 
 use Sylius\Component\Core\Exception\HandleException;
 
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
  */
-interface LocaleChangeHandlerInterface
+interface CodeChangeHandlerInterface
```

Comment:
I agree with @Arminek. These are two separate concepts and we shouldn't merge them only because the implementation is now the same and the code is duplicated.

Quoting [good explanation of DRY](http://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests):

> **DRY (Don't repeat yourself)** promotes the orthogonality of the code.

> Removing duplication ensures that every concept in the system has a single authoritative representation in the code. A change to a single business concept results in a single change to the code. DRY increases maintainability by isolating change (risk) to only those parts of the system that must change.